[2024-04-12T07:19:41.089+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-12T07:19:41.350+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_complex.lookup_entry_result manual__2024-04-12T07:19:14.791350+00:00 [queued]>
[2024-04-12T07:19:41.504+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_complex.lookup_entry_result manual__2024-04-12T07:19:14.791350+00:00 [queued]>
[2024-04-12T07:19:41.507+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-12T07:19:41.540+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): lookup_entry_result> on 2024-04-12 07:19:14.791350+00:00
[2024-04-12T07:19:41.602+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4028) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-12T07:19:41.606+0000] {standard_task_runner.py:63} INFO - Started process 4146 to run task
[2024-04-12T07:19:41.600+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_complex', 'lookup_entry_result', 'manual__2024-04-12T07:19:14.791350+00:00', '--job-id', '20', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_complex.py', '--cfg-path', '/tmp/tmpqyk8ph5w']
[2024-04-12T07:19:41.611+0000] {standard_task_runner.py:91} INFO - Job 20: Subtask lookup_entry_result
[2024-04-12T07:19:42.061+0000] {task_command.py:426} INFO - Running <TaskInstance: example_complex.lookup_entry_result manual__2024-04-12T07:19:14.791350+00:00 [running]> on host 48c2eff06d44
[2024-04-12T07:19:42.490+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_complex' AIRFLOW_CTX_TASK_ID='lookup_entry_result' AIRFLOW_CTX_EXECUTION_DATE='2024-04-12T07:19:14.791350+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-12T07:19:14.791350+00:00'
[2024-04-12T07:19:42.493+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-12T07:19:42.497+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-12T07:19:42.501+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo lookup_entry_result']
[2024-04-12T07:19:42.570+0000] {subprocess.py:86} INFO - Output:
[2024-04-12T07:19:42.579+0000] {subprocess.py:93} INFO - lookup_entry_result
[2024-04-12T07:19:42.585+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-12T07:19:42.586+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-12T07:19:42.640+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=lookup_entry_result, execution_date=20240412T071914, start_date=20240412T071941, end_date=20240412T071942
[2024-04-12T07:19:42.747+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-12T07:19:42.793+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-12T07:19:42.813+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
